tests/block_tools.py

```diff
@@ -2057,9 +2096,8 @@ def create_block_tools(
     print(f"  create_block_tools called {create_block_tools_count} times")
     bt = BlockTools(constants, root_path, const_dict, keychain, config_overrides=config_overrides)
 
-    asyncio.get_event_loop().run_until_complete(bt.setup_keys())
-    asyncio.get_event_loop().run_until_complete(bt.setup_plots())
-
+    asyncio.run(bt.setup_keys())
```

Comment:
let's revert this, I remember that this caused an issue before, but not exactly sure why.